src/main/java/teammates/ui/webapi/GetFeedbackSessionLogsAction.java

```diff
@@ -30,7 +49,74 @@ void checkSpecificAccessControl() {
     }
 
     @Override
-    ActionResult execute() {
-        return null;
+    JsonResult execute() {
+        String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
+        if (logic.getCourse(courseId) == null) {
+            return new JsonResult("Course not found", HttpStatus.SC_NOT_FOUND);
+        }
+        String email = getRequestParamValue(Const.ParamsNames.STUDENT_EMAIL);
```

Comment:
Again same concern here about the registration key vs email

Reply:
Similar to above